.scalafix.conf

```diff
@@ -1 +1,5 @@
+include ".scalafix-common.conf"
+
+rules = [] # Disable organize imports for now
```

Comment:
The organize imports was breaking the tests with ambiguous implicits for arbitraries. I guess some of them are sensitive to the import order which is a bit strange.